src/styles.css

```diff
@@ -1,4 +1,4 @@
-:root {
+/* :root {
```

Comment:
is this file used? If not used please delete. Only keep files that are actually used in this PR.

Reply:
unused file. deleted